project/Build.scala

```diff
@@ -1036,7 +1062,7 @@ object Build extends sbt.Build {
   lazy val partest: Project = Project(
       id = "partest",
       base = file("partest"),
-      settings = commonSettings ++ Seq(
+      settings = commonSettings ++ fatalWarningsSettings ++ Seq(
```

Comment:
If you manage to turn it off, OK. But I think I tried to, and sometimes it doesn't really turn it off even if you ask for it.